### ISSUE TITLE
Improved Statistics

### ISSUE THREAD

**Issue by Caranxnet** [cubecart/v6#3639]
Good morning,

this is about the Control Panel stats: It would be very useful if we could select a period of time for each statistic available. For example, the bestseller products, what we have now is a list starting on the Jurassic and is not of great use. If we could pick, let's say January the 1st 2022, it would give me a way better picture of the current situation. If I am running a campaign, this is a great tool to see some of the results.

Don't know if this is doable, thanks for your attention anyway

regards

Nicola

**Comment by ma-ximus** [comment]
Yes, this is definitely a sorely needed feature. Right now, I copy paste the best selling stats to an excel sheet at the end of every year, then compute the difference to see what was sold within a year.
